src-doc/contenuti/processi-di-supporto/documentazione.tex

```diff
@@ -24,6 +24,10 @@ \subsubsection{Ciclo di vita dei verbali}
 
 I verbali seguono lo stesso ciclo di vita dei docuementi, ma non avranno fasi di ripetizione, in quanto verranno redatti, verificati e rilasciati solamente una volta. Un verbale non sarà infatti un documento modificabile.
 
+\subsubsection{Ciclo di vita della documentazione sviluppatori microservizio WebApp}
+
+La documentazione per gli sviluppatori riguardante il microservizio della WebApp segue lo stesso ciclo di vita dei documenti, senza però fasi di ripetizione, in quanto verrà redatta, verificata e rilasciata una volta sola dopo che la WebApp sarà stata terminata e approvata.
```

Comment:
É giusta questa cosa? Ne avevamo già discusso in questo modo? Ne ho perso memoria. Per il resto però mi sembra bene e approvo.

Reply:
Non ho memoria della discussione di questa cosa quindi sono andato un po' a intuito. Se ho sbagliato correggo